setup.py

```diff
@@ -32,8 +32,12 @@ def get_long_description() -> str:
     package_dir={"": "src"},
     include_package_data=True,
     zip_safe=False,
-    install_requires=["anyio==1.*", "typing-extensions==3.7.*; python_version<'3.8'"],
-    python_requires=">=3.7",
+    install_requires=[
+        "anyio~=3.2",
```

Comment:
Does this mean what I think it does, i.e. "3.2+, but below 4.0"?

Reply:
Yep